libcudacxx/include/cuda/std/detail/libcxx/include/bitset

```diff
@@ -129,932 +137,889 @@ template <size_t N> struct hash<std::bitset<N>>;
 #  pragma system_header
 #endif // no system header
 
-_LIBCUDACXX_PUSH_MACROS
-#include <__undef_macros>
+#include <cuda/std/__algorithm/count.h>
+#include <cuda/std/__algorithm/fill.h>
+#include <cuda/std/__algorithm/find.h>
+#include <cuda/std/detail/libcxx/include/__assert> // all public C++ headers provide the assertion handler
+#include <cuda/std/detail/libcxx/include/__bit_reference>
+#include <cuda/std/__functional/hash.h>
+#include <cuda/std/__functional/unary_function.h>
+#include <cuda/std/detail/libcxx/include/__string>
```

Comment:
I would strongly favor of putting this into a `__string` subfolder and name it `char_traits.h`

Reply:
Did we get rid of all the cases in packaging where turning a file into a directory would be potentially problematic at this point?